Library/Formula/gr-osmosdr.rb

```diff
@@ -0,0 +1,88 @@
+class GrOsmosdr < Formula
+  desc "Osmocom GNU Radio blocks"
+  homepage "https://osmocom.org/projects/sdr/wiki/GrOsmoSDR"
+  url "http://cgit.osmocom.org/gr-osmosdr/snapshot/gr-osmosdr-0.1.4.tar.xz"
+  sha256 "1945d0d98fd4b600cb082970267ec2041528f13150422419cbd7febe2b622721"
+  head "git://git.osmocom.org/gr-osmosdr"
+
+  depends_on "cmake" => :build
+  depends_on "gnuradio"
+  depends_on "librtlsdr" => :recommended
+  depends_on :python => :recommended
+  depends_on "hackrf" => :optional
+  depends_on "libbladerf" => :optional
+  depends_on "uhd" => :optional
+  depends_on "airspy" => :optional
+
+  patch :DATA
```

Comment:
Nope, because it is Homebrew specific (the Python "-undefined dynamic_lookup" thingy which the audit complains if it is linked to a specific Framework version). At least I think it is specific to Homebrew... It woould work totally fine when linking statically to a Framework Python, but then brew audit complains. The formula for gnuradio does the same thing (that's where I got it from).